packages/lesswrong/components/linkPreview/HoverPreviewLink.jsx

```diff
@@ -1,34 +1,8 @@
 import React from 'react';
-import { Components, registerComponent, parseRoute, Utils } from 'meteor/vulcan:core';
+import { Components, registerComponent, parseRoute, parsePath, Utils } from 'meteor/vulcan:core';
```

Comment:
FYI, this particular commit was complicated enough that I bounced off finishing the review, which otherwise I'd have been able to easily approve in one sitting. I'd recommend splitting this sort of fix into it's own PR.